Mathlib/Tactic/Linarith/Oracle/FourierMotzkin.lean

```diff
@@ -32,7 +32,29 @@ we conclude that the original system was unsatisfiable.
 -/
 
 open Batteries
-open Std (format ToFormat)
+open Std (format ToFormat TreeSet)
+
+namespace Std.TreeSet
```

Comment:
Can this be upstreamed to `Std` already?
(If so, filing such a PR and leaving an adaptation note would be great!)